app/content/profile/index.js

```diff
@@ -264,35 +258,14 @@ function Index() {
             </>
           )}
 
-          <BottomSheetModal
+          <BottomSheetWrapper
```

Comment:
When testing on android, I found that if I went to edit the name or password, it expanded the modal to the whole screen and the close button was behind the top section of the OS and I was unable to click it. I have attached screenshots of this.
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/41858c5c-2564-40bf-aaec-2ac9420fbfb5)
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/5b1310e1-3e75-4482-8873-9611fb614700)

Reply:
I think we'll have to revert to snappoints for this. Nothing I've tried works and I think enableDynamicSizing's behavior is really buggy when the keyboard is involved